pkg/devspace/config/loader/validate.go

```diff
@@ -377,8 +381,8 @@ func validateDev(config *latest.Config) error {
 	if config.Dev.Ports != nil {
 		for index, port := range config.Dev.Ports {
 			// Validate imageName and label selector
-			if len(port.LabelSelector) == 0 && port.ImageSelector == "" {
-				return errors.Errorf("Error in config: image selector and label selector are nil in ports config at index %d", index)
+			if port.ContainerName != "" && len(port.LabelSelector) == 0 && port.ImageSelector == "" {
```

Comment:
These need to be separte checks:
1. containerName != "" && len(labelSelector) == 0
2. len(labelSelector) == 0 && imageSelector == ""